src/k2/dto/ControlPlaneOracle.h

```diff
@@ -139,6 +139,18 @@ struct CreateSchemaResponse {
     K2_DEF_FMT(CreateSchemaResponse);
 };
 
+static constexpr int64_t ANY_VERSION = -1;
```

Comment:
I think this needs to be also `inline`

Reply:
Changed. However it was compiling fine without it.